api/json-vs-xml.md

```diff
@@ -1,31 +1,120 @@
 ---
 description: >-
-  Choose between JSON and XML in the `api-connector` and `order-webhook-connector`
+  Choose between JSON and XML formats in the Tradecloud API. Learn about format support, when to use each, and how to implement both JSON and tXML formats.
 ---
 
 # JSON versus XML
 
-The Tradecloud API by default supports a proprietary JSON format, but for some API endpoints a proprietary XML format is supported.
+The Tradecloud API supports both JSON and XML formats to accommodate different integration requirements and legacy systems. This guide helps you understand the differences and choose the appropriate format for your implementation.
 
-## JSON
+## Overview
 
-The Tradecloud API by default supports a proprietary Tradecloud [JSON format](requests.md#json-body). This is the default for all endpoints.
+The Tradecloud API provides two data formats:
 
-## XML
+- **JSON**: The default format for all API endpoints
+- **tXML**: A proprietary XML format available for specific endpoints
 
-The Tradecloud API supports a proprietary Tradecloud [XML format](requests.md#xml-body), called `tXML`, which is a 1-on-1 translation of the proprietary JSON format:
+Both formats contain identical data structures and functionality - tXML is a one-to-one mapping of the JSON format.
 
-* [Send order](https://swagger-ui.accp.tradecloud1.com/?url=https://api.accp.tradecloud1.com/v2/api-connector/specs.yaml#/buyer-endpoints/sendOrderByBuyerRoute) and
-* [Send single delivery order](https://swagger-ui.accp.tradecloud1.com/?url=https://api.accp.tradecloud1.com/v2/api-connector/specs.yaml#/buyer-endpoints/sendSingleDeliveryOrderByBuyerRoute) endpoints support `tXML`.
+## When to use each format
 
-You can see an XML example by selecting "application/xml" in the "Parameter content type" dropdown, under the "Example Value" in above API specifications:
+### Use JSON when
+
+- Building new modern integrations (recommended)
+- Implementing REST API clients
+- You need smaller payload sizes
```

Comment:
Although this is a pre of JSON compared to XML, I think this is hardly ever a requirement of customers. imo this bullet can be removed.

Reply:
removed